src/schemas/quantity.test.js

```diff
@@ -0,0 +1,217 @@
+import { quantitySchema } from './quantity.js'
+
+describe('Quantity Schema Validation', () => {
+  describe('isEstimate Field Validation', () => {
+    describe('Valid isEstimate Values', () => {
+      it('should accept isEstimate: true', () => {
+        const validPayload = {
+          metric: 'Tonnes',
+          amount: 10,
+          isEstimate: true
+        }
+
+        const { error } = quantitySchema.validate(validPayload)
+        expect(error).toBeUndefined()
+      })
+
+      it('should accept isEstimate: false', () => {
+        const validPayload = {
+          metric: 'Tonnes',
+          amount: 10,
+          isEstimate: false
+        }
+
+        const { error } = quantitySchema.validate(validPayload)
+        expect(error).toBeUndefined()
+      })
+
+      // Joi automatically converts string "true"/"false" to boolean, so these are actually valid
+      it('should accept isEstimate: "true" (string) - Joi auto-converts', () => {
+        const validPayload = {
+          metric: 'Tonnes',
+          amount: 10,
+          isEstimate: 'true'
+        }
+
+        const { error } = quantitySchema.validate(validPayload)
+        expect(error).toBeUndefined()
+      })
+
+      it('should accept isEstimate: "false" (string) - Joi auto-converts', () => {
+        const validPayload = {
+          metric: 'Tonnes',
+          amount: 10,
+          isEstimate: 'false'
+        }
+
+        const { error } = quantitySchema.validate(validPayload)
+        expect(error).toBeUndefined()
+      })
+    })
+
+    describe('Invalid isEstimate Values', () => {
+      it('should reject isEstimate: 1 (number)', () => {
+        const invalidPayload = {
+          metric: 'Tonnes',
+          amount: 10,
+          isEstimate: 1
+        }
+
+        const { error } = quantitySchema.validate(invalidPayload)
+        expect(error).toBeDefined()
+        expect(error.details[0].message).toContain(
```

Comment:
shouldn't it be asserting if the message error is coming for the isEstimated field? as it does in the last test in the file?